### PR TITLE
sellers yml and reviews yml

### PR DESCRIPTION
**Pull request by TangoIndiaMango**
Hi, i have sellers delete.yml and all the required yml files for the reviews ready for review.

### CODE REVIEW

**Comment by samdoghor** [comment]
some files for the review are still missing, (read_all.yml and read_one.yml). Also, complete the sellers docs so review can be once.

Thank you.

**Comment by samdoghor** [comment]
You didn't uncomment the swag decorator in the resource files.

**Comment by TangoIndiaMango** [comment]
hi please i made a new push, do review. Thank you

File: src/swagger/review/delete.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+title: Delete a review
+description: Delete a review
+tags:
+  - Reiview
```
Comment (samdoghor) [inline src/swagger/review/delete.yml:4]: Reviews

File: src/swagger/review/get_all.yml [hunk 2]
```diff
@@ -1,41 +1,67 @@
...
-    description: the products id for a review
-  - name: created_at
-    in: query
-    type: datetime
-    description: the date the review was created
-  - name: updated_at
-    in: query
-    type: datetime
-    description: the date the review was updated
+  - Review
```
Comment (samdoghor) [inline src/swagger/review/get_all.yml:4]: Reviews

File: src/swagger/review/post.yml [hunk 3]
```diff
@@ -0,0 +1,88 @@
+title: Create a new review
+description: Create a new review
+tags:
+  - Review
```
Comment (samdoghor) [inline src/swagger/review/post.yml:4]: Reviews

File: src/swagger/review/put.yml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        required:
+          - comment
+          - images
+          - sellers_id
+          - products_id
+  required: true
+
+responses:
+  200:
+    description: The product was Updated successfully
```
Comment (samdoghor) [inline src/swagger/review/put.yml:42]: The review not product

File: src/swagger/seller/get_all.yml [hunk 5]
```diff
@@ -1,25 +1,76 @@
...
+      application/json:
+        schema:
+          type: array
+          items:
+            type: object
+            properties:
+              first_name:
+                type: string
+                description: The seller's first name
+                last_name:
```
Comment (samdoghor) [inline src/swagger/seller/get_all.yml:20]: Indent to be on the same line with first_name

File: src/swagger/seller/get_all.yml [hunk 6]
```diff
@@ -1,25 +1,76 @@
...
+                type: string
+                description: The seller's last name
+                email:
```
Comment (samdoghor) [inline src/swagger/seller/get_all.yml:23]: Indent to be on the same line with first_name

File: src/swagger/seller/get_all.yml [hunk 7]
```diff
@@ -1,25 +1,76 @@
...
+                type: string
+                description: The seller's email
+                phone:
```
Comment (samdoghor) [inline src/swagger/seller/get_all.yml:26]: Indent to be on the same line with first_name

File: src/swagger/seller/get_all.yml [hunk 8]
```diff
@@ -1,25 +1,76 @@
...
+                type: string
+                description: The seller's phone number
+                username:
```
Comment (samdoghor) [inline src/swagger/seller/get_all.yml:29]: Indent to be on the same line with first_name

File: src/swagger/seller/get_all.yml [hunk 9]
```diff
@@ -1,25 +1,76 @@
...
+                type: string
+                description: The seller's username
+                rating:
```
Comment (samdoghor) [inline src/swagger/seller/get_all.yml:32]: Indent to be on the same line with first_name

File: src/swagger/seller/get_all.yml [hunk 10]
```diff
@@ -1,25 +1,76 @@
...
+                  { 
+                    "last_name": Doe, 
+                    "first_name": John, 
+                    "email": admin@admin.com,
+                    "phone": +234819566023
+                    "username": "DoeSeller"
+                    "rating": 25
+                  },
+                  { 
+                    "last_name": "Roe", "first_name": "Jerry",
```
Comment (samdoghor) [inline src/swagger/seller/get_all.yml:48]: To maintain same format across all docs bring first name down

File: src/swagger/seller/post.yml [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+          - last_name
+          - phone
+          - username
+          - email
+          - password
+  required: true
+
+responses:
+  200:
+    description: The product was created successfully
```
Comment (samdoghor) [inline src/swagger/seller/post.yml:43]: Seller not product

File: src/swagger/seller/put.yml
Comment (samdoghor) [inline src/swagger/seller/put.yml:1]: Email was not included as seen in others

**Review by samdoghor — CHANGES_REQUESTED** [review]

File: src/swagger/seller/put.yml
Comment (TangoIndiaMango) [reply src/swagger/seller/put.yml:1]: image



It's not needed in the update method

**Review by samdoghor — APPROVED** [review]

**Pull request merged**